src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -108,6 +129,8 @@ object EasyStageDataset {
       Try { dir.listFiles().toList }.flatMap(_.map(visit).collectResults.map(_ => ()))
     }
 
+    def getBagRelativePath(path: Path) = s.bagitDir.toPath.relativize(path)
```

Comment:
Can do, although I am not sure what the best practice is for nested functions. Note that we are inside a monster-function that nests a lot of utility functions. Not sure if that is a good approach (but that is something for another day).